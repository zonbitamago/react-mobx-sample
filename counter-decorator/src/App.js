import React, { Component } from "react";
import logo from "./logo.svg";
import "./App.css";
import ShowCount from "./components/ShowCount/ShowCount";
const App = class App extends Component {
  constructor(props) {
    super(props);
  }
  render() {
    return (
      <div className="App">
        <ShowCount store={this.props.store} />
      </div>
    );
  }
};

export default App;
