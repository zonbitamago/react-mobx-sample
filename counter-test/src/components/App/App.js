import React, { Component } from "react";
import PropTypes from "prop-types";
import styles from "./App.css";

class App extends Component {
  constructor(props) {
    super(props);
  }

  render() {
    return (
      <div className="App">
        <h2>Hello React World!</h2>
      </div>
    );
  }
}

App.propTypes = {};

App.defaultProps = {};

export default App;
