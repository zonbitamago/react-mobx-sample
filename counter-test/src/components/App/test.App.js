import React from 'react'
import { shallow } from 'enzyme'

import App from './App'

describe('App', () => {
  let component, props

  beforeEach(() => {
    props = {}
    component = shallow(<App {...props} />)
  })

  it('should', () => {
    expect(component).toMatchSnapshot()
  })
})