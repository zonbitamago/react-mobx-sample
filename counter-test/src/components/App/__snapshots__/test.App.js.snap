// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2>
          Hello React World!
        </h2>,
        <Counter
          store={undefined}
        />,
      ],
      "className": "App",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Hello React World!",
        },
        "ref": null,
        "rendered": "Hello React World!",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "store": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2>
            Hello React World!
          </h2>,
          <Counter
            store={undefined}
          />,
        ],
        "className": "App",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Hello React World!",
          },
          "ref": null,
          "rendered": "Hello React World!",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "store": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
